src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -149,23 +149,25 @@ export default {
 		},
 	},
 	mounted() {
-		this.apollo.query({
-			query: privateLendMenuQuery,
-			variables: {
-				userId: this.userId,
-			}
-		}).then(({ data, errors }) => {
-			if (!errors) {
-				this.favoritesCount = _get(data, 'lend.loans.totalCount');
-				this.savedSearches = _get(data, 'my.savedSearches.values');
-			} else {
-				this.favoritesCount = 0;
-				this.savedSearches = [];
-			}
-		}).finally(() => {
-			// data might have changed since the initial render, so trigger any needed updates
-			this.onOpen();
-		});
+		if (this.hasUserId) {
```

Comment:
This query was being called every time the page loaded, even when the user was not logged in and `this.userId` was undefined/null. On the vm this query is fairly slow, even without a logged in user, so hopefully skipping this when it doesn't have a user id will speed things up while developing and slightly speed things up in prod.